Topten.RichTextKit/TextShaping/TextShaper.cs

```diff
@@ -65,10 +64,10 @@ private TextShaper(SKTypeface typeface)
 
             // Load the typeface stream to a HarfBuzz font
             int index;
-            using (var blob = typeface.OpenStream(out index).ToHarfBuzzBlob())
+            using (var blob = GetHarfBuzzBlob(typeface.OpenStream(out index)))
             using (var face = new Face(blob, (uint)index))
             {
-                face.UnitsPerEm = (uint)typeface.UnitsPerEm;
+                face.UnitsPerEm = typeface.UnitsPerEm;
```

Comment:
Unfortunately a breaking change with the v2.x series. We decided to break this since we are now doing the binding properly with v2.x.

https://github.com/mono/SkiaSharp/blob/master/changelogs/HarfBuzzSharp/2.6.1/HarfBuzzSharp.breaking.md